mutt.h

```diff
@@ -26,9 +26,6 @@
 #ifdef HAVE_UNISTD_H
 # include <unistd.h> /* needed for SEEK_SET */
 #endif
-#ifdef HAVE_UNIX_H
-# include <unix.h>   /* needed for snprintf on QNX. */
```

Comment:
I searched through the docs of QNX, and found that ① they're C99 compliant and ② there's a [`stdio.h` featuring snprintf](http://www.qnx.com/developers/docs/6.5.0SP1.update/index.html#./com.qnx.doc.dinkum_en_c99/stdio.html).

So I vote for removing those useless three lines.